src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -17381,6 +17460,79 @@ public static void Main()
         }
 
         [Fact]
+        public void OverriddenPropertyWithDifferentTupleNamesInReturn()
```

Comment:
Yes, we should have a test.  It was discussed at LDM and we care only about the "most derived" base. 
Similarly to the named parameters - most derived names are used.

---
In reply to: [81779500](https://github.com/dotnet/roslyn/pull/14229#discussion_r81779500) [](ancestors = 81779500)